scania-digital-design-system/sdds
# AB#295 checkboxes styling

Added checkbox styling
- Explore animation on click
- Add checkbox

How to test:

- Check Figma >> UI Kit (Global) >> Checkbox

Fix: [AB#295](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/295)
